### DISCUSSION TITLE
Inability to add new todo items in Project Board and project table view. This seems to be the only real feature I would need from a GitHub mobile app is to be able to update the planning on the go. But this key feature is not there. Can’t scroll down on board view and no way to add a new item

### DISCUSSION THREAD

**Discussion by DennisBack** [community/community#78921] [Mobile] [answered]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.142.0 (117666737)
Device: iPhone15,3 (iOS 17.1.2)
</details>

**Comment by stevepopovich** [comment, marked as answer]
Hi @DennisBack sounds like you are having trouble, sorry about that. As far as we know, you should be able to scroll through project boards. We don't fully support boards, but they should render as a list.

We don't have the ability to create an Issue or Draft in projects currently, we have that in the backlog. You can, however, create an issue in a repo and then edit the properties to add it to a project. I acknowledge this is a bit of workaround. I will log your feedback. Thanks for writing in!

**Reply by chopov1** [reply]
Would love to see this implemented as well. Kind of the main reason I would want GitHub on mobile. Glad you explained that it is not supported yet so I didn't think I was crazy.👍🏼 thank you for the work around as well.
